Library/Homebrew/rubocops/livecheck.rb

```diff
@@ -199,6 +247,14 @@ class LivecheckRegexCaseInsensitive < FormulaCop
 
         MSG = "Regexes should be case-insensitive unless sensitivity is explicitly required for proper matching."
 
+        sig {
+          override.params(
+            _node:              RuboCop::AST::ClassNode,
+            _class_node:        RuboCop::AST::ConstNode,
+            _parent_class_node: RuboCop::AST::ConstNode,
+            body_node:          RuboCop::AST::Node,
+          ).void
+        }
```

Comment:
Oof, this `sig` is a bit fatiguing, it may be the best argument in favor of https://github.com/sorbet/sorbet/issues/2286 that I've come across.

🤔 I wonder if we could create a T::Struct or such for these params and pass that around?

(PR looks great, just thinking aloud here.)

Reply:
Ha, while working on that file I was thinking, "Type aliases are a thing, why not signature aliases?", so I'm right there with you.

> I wonder if we could create a T::Struct or such for these params and pass that around?

Not too long ago I was tinkering with `utils/curl` (in relation to livecheck) and trying to figure out how to pass around args to `#curl_args` without needing explicit args and a ton of verbose type signatures everywhere. I was going down a similar path when I last left it, so it's worth trying out here, too. I'll try to tinker with it today to see how it goes but feel free to beat me to it if that doesn't pan out.

We have ~63 `audit_formula` methods, so it's probably best to drop the `rubocops/livecheck.rb` sigs here in favor of a better approach for all the `FormulaCop`s in a follow-up PR. I was hoping for this kind of feedback, so thanks for that.